docs/tutorial/file.rst

```diff
@@ -148,6 +151,40 @@ Once you access the raw data, those are regular arrays which use regular array i
     channel_slice = file.force1x['1.5s':'20s']  # timestamps
     data_slice = file.force1x.data[20:40]  # indices into the array
 
+Downsampling
+^^^^^^^^^^^^
+
+A slice can be downsampled using various methods.
+
+To downsample to a specific frequency use `downsampled_to` with the desired frequency in Hz::
+
+    channel = file.force1x # original frequency 78125 Hz
+    timestep = np.diff(channel.timestamps[:2]) * 1e-9        # timestep 12.8 us
+
+    ds_channel = channel.downsampled_to(3125)
+    ds_timestep = np.diff(ds_channel.timestamps[:2]) * 1e-9  # timestep 320 us
+
+This method can only be applied successfully if the ratio of the current and target frequencies
+is an integer. To automatically find the nearest higher frequency that will fulfill this requirement,
+use the `method="ceil"`::
+
+    ds_channel2 = channel.downsampled_to(3126, method="ceil")
+    ds_timestep2 = np.diff(ds_channel2.timestamps[:2]) * 1e-9  # timestep 307.2 us
+
+For data that is recorded with variable sampling frequencies, use `method="force"`.
```

Comment:
`For data that is recorded with variable sampling frequencies, it is usually not possible to downsample to a single sample rate, while maintaining an equal number of samples per downsampled sample. To force downsampling to a single frequency in the case of variable sample rates, use method="force"::`

then the code example. And the maybe something to mention that this same flag can also be used to force the downsampling rate in the other case:

`Note that this same flag can also be used to force a specific downsampling rate for non-integer downsampling rates.`